rotkehlchen/tests/conftest.py

```diff
@@ -110,3 +112,12 @@ def profiler(request):
 
     if profiler_instance is not None:
         profiler_instance.stop()
+
+
+def requires_env(allowed_envs: List[str]):
```

Comment:
We need a way to specify multiple environments easily.

Specifically what I want to be able to achieve is to run if:
- It's the nightlies run (we have to then make sure that it can take more than 60 minutes!!! -- and not time out at 60)
- Or the commit contains a tag. Say [run-all-tests].